src/routes/backup/ShowBackupLabels.js

```diff
@@ -23,7 +23,7 @@ const modal = ({
   }
 
   const kubernetesStatus = item && item.KubernetesStatus ? JSON.parse(item.KubernetesStatus) : {}
-  const snapshotCreated = item && item.snapshotCreated ? item.snapshotCreated : ''
+  const snapshotCreatedAt = item && item.snapshotCreated ? item.snapshotCreated : ''
```

Comment:
could you please revert this file change?

Reply:
https://github.com/longhorn/longhorn-ui/blob/75cae8cb7bde296716e222e21a348ed697b9e902/src/routes/backup/ShowBackupLabels.js#L60

The attribute is still snapshotCreated onle the variable name has been changed to snapshotCreatedAt. It will be displayed normally. I'll change it later. thanks